frontend/src/components/EmployeeDetails.js

```diff
@@ -56,6 +59,19 @@ const EmployeeDetails = ({employee}) => {
         setShowModal(true);
     }
         
+    useEffect(() => {
+        const fetchData = async () => {   
+            const response = await fetch('/api/clinics', {
+                headers: {
+                  'Authorization': `Bearer ${user.token}`
```

Comment:
Could you put here also conte-type as applicaton/json

Reply:
Content-type added